test_scripts/Resumption/InteriorVehicleData/024_InteriorVD_resumption_repeated_subscription.lua

```diff
@@ -0,0 +1,69 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal:
```

Comment:
@GetmanetsIrina Please add link to proposal as in other scripts for this feature